org.tryton.Tryton-74.yml

```diff
@@ -0,0 +1,35 @@
+id: org.tryton.Tryton-74
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '46'
```

Comment:
47 is latest

Reply:
goocanvas does not compile on it.

Reply:
I found a patch to build goocanvas on 47 but now I have this error:
```
Running appstream-compose
bwrap: execvp appstream-compose: No such file or directory
Error: ERROR: appstream-compose failed: Child process exited with code 1
```
while the same manifest works with 46.

Reply:
use org.flatpak.Builder from the docs linked in PR.